UM/Trust.py

```diff
@@ -285,6 +285,16 @@ def removeCached(path: str) -> bool:
             Logger.logException("e", "Removal of pycache for unbundled path '{0}' failed.".format(path))
         return False
 
+    @staticmethod
+    def isPathInLocation(location: str, path: str) -> bool:
```

Comment:
It's a public function now, so i think this has more need for documentation now.

Reply:
Fair enough!

Reply:
Fixed in https://github.com/Ultimaker/Uranium/pull/717/commits/81250f7af3c809cf213a6464016698bfeb9c0c12